ingest/icu/tensorizer.py

```diff
@@ -348,6 +349,14 @@ def _copy_hd5(staging_dir, destination_dir, file):
 
 
 def tensorize(args):
+
+    if not os.path.isdir(args.bedmaster) or len(os.listdir(args.bedmaster)) == 0:
+        raise ValueError(
+            f"CDAC server with Bedmaster data is not mounted in {args.bedmaster}",
```

Comment:
Generalize beyond CDAC:
```suggestion
            f"Server with Bedmaster data is not mounted in {args.bedmaster}",
```